contrib/graph-gophers/graphql-go/graphql.go

```diff
@@ -25,9 +25,10 @@ import (
 )
 
 const (
-	tagGraphqlField = "graphql.field"
-	tagGraphqlQuery = "graphql.query"
-	tagGraphqlType  = "graphql.type"
+	tagGraphqlField     = "graphql.field"
+	tagGraphqlQuery     = "graphql.query"
+	tagGraphqlType      = "graphql.type"
+	tagGraphqlOperation = "graphql.operation"
```

Comment:
I think for this we want to have `tagGraphqlOperation = "graphql.operation.name"` to be consistent with our JS graphql integration: https://github.com/DataDog/dd-trace-js/blob/aafacd0200b618024af4066b0a1475c0f1e4c55c/packages/datadog-plugin-graphql/src/index.js#L241-L247

Reply:
Done.